examples/contact-sensor-app/linux/README.md

```diff
@@ -0,0 +1,146 @@
+
+    Use specific bluetooth interface for BLE advertisement and connections.
+
+    `interface id`: the number after `hci` when listing BLE interfaces by
+    `hciconfig` command, for example, `--ble-device 1` means using `hci1`
+    interface. Default: `0`.
+
+## Running the Complete Example on Raspberry Pi 4
+
+> If you want to test Echo protocol, please enable Echo handler
+>
+>     gn gen out/debug --args='chip_app_use_echo=true'
+>     ninja -C out/debug
+
+-   Prerequisites
+
+    1. A Raspberry Pi 4 board
+    2. A USB Bluetooth Dongle, Ubuntu desktop will send Bluetooth advertisement,
+       which will block CHIP from connecting via BLE. On Ubuntu server, you need
+       to install `pi-bluetooth` via APT.
+    3. Ubuntu 20.04 or newer image for ARM64 platform.
+
+-   Building
+
+    Follow [Building](#building) section of this document.
+
+-   Running
+
+    -   [Optional] Plug USB Bluetooth dongle
+
+        -   Plug USB Bluetooth dongle and find its bluetooth device number. The
+            number after `hci` is the bluetooth device number, `1` in this
+            example.
+
+                  $ hciconfig
+                  hci1:	Type: Primary  Bus: USB
+                      BD Address: 00:1A:7D:AA:BB:CC  ACL MTU: 310:10  SCO MTU: 64:8
+                      UP RUNNING PSCAN ISCAN
+                      RX bytes:20942 acl:1023 sco:0 events:1140 errors:0
+                      TX bytes:16559 acl:1011 sco:0 commands:121 errors:0
+
+                  hci0:	Type: Primary  Bus: UART
+                      BD Address: B8:27:EB:AA:BB:CC  ACL MTU: 1021:8  SCO MTU: 64:1
+                      UP RUNNING PSCAN ISCAN
+                      RX bytes:8609495 acl:14 sco:0 events:217484 errors:0
+                      TX bytes:92185 acl:20 sco:0 commands:5259 errors:0
+
+        -   Run Linux Contact Sensor App
+
+                  $ cd ~/connectedhomeip/examples/contact-sensor-app/linux
+                  $ sudo out/debug/chip-contact-sensor-app --ble-device [bluetooth device number]
+                  # In this example, the device we want to use is hci1
+                  $ sudo out/debug/chip-contact-sensor-app --ble-device 1
+
+        -   Test the device using ChipDeviceController on your laptop /
+            workstation etc.
+
+## Running RPC Console
+
+-   As part of building the example with RPCs enabled the chip_rpc python
+    interactive console is installed into your venv. The python wheel files are
+    also created in the output folder: out/debug/chip_rpc_console_wheels. To
+    install the wheel files without rebuilding:
+    `pip3 install out/debug/chip_rpc_console_wheels/*.whl`
+
+-   To use the chip-rpc console after it has been installed run:
+    `chip-console -s localhost:33000 -o /<YourFolder>/pw_log.out`
+
+-   Then you can Get and Set the light using the RPCs:
+    `rpcs.chip.rpc.Lighting.Get()`
+
+    `rpcs.chip.rpc.Lighting.Set(on=True, level=128, color=protos.chip.rpc.LightingColor(hue=5, saturation=5))`
```

Comment:
Can you make this a relevant example for the contact sensor?

Reply:
Yes. I forgot to correct this one.

Reply:
@agners Where can I find the pigweed documentation for Matter with the list of available methods?

Reply:
I am not very familiar with the Pigweed RCP things, but it seems this is from `examples/common/pigweed/`. Maybe there is autocomplete? (along with "tab-tab" to get  a list of available rpcs?).

Reply:
Fixed by [8e7f35a](https://github.com/lboue/connectedhomeip/commit/8e7f35adba459e04cd6603f09142e5a503b36aa5)